prReview.js

```diff
@@ -0,0 +1,7 @@
+function findFolderByName(n) {
+	try {
+		return n === null ? null : (Folder.find({ name: n }) || Folder.create({ name: n }));
+	} catch(e) {
+		return null;
```

Comment:
**chores _(blocking)_:** As a good practise, returning null should be avoided as much as possible. Kindly check out [Throw exceptions instead of returning null](https://www.code4it.dev/cleancodetips/exceptions-instead-of-null).
Besides, we could do better in handling this errors. Kindly check out [Handling errors in javascript](https://www.javascriptinstitute.org/javascript-tutorial/handling-errors-in-javascript/)

Reply:
Let's also add some tests to improve code quality.

Well done.